### PR TITLE
Fix #55: Retry aiohttp requests

### LINKED ISSUE #55: Retry requests made with aiohttp
Currently, only requests made with `KintoClient` are retried

### PR DESCRIPTION
**Pull request by leplatrem**
Fix #55

### CODE REVIEW

**Comment by leplatrem** [comment]
@glasserc As you said elsewhere, you're not in favor of retries. I kept them mainly because I imported them from our lambda code. I think it makes sense to retry, especially if we set a small timeout.

File: checks/core/cloudfront_age.py [hunk 1]
```diff
@@ -3,23 +3,17 @@
...
-REQUESTS_TIMEOUT_SECONDS = int(os.getenv("REQUESTS_TIMEOUT_SECONDS", 5))
-
 
 async def run(url: str, max_age: str) -> CheckResult:
     age = 0
 
-    timeout = aiohttp.ClientTimeout(total=REQUESTS_TIMEOUT_SECONDS)
-    async with aiohttp.ClientSession(timeout=timeout) as session:
+    async with ClientSession() as session:
         async with session.head(url) as response:
```
Comment (glasserc) [inline checks/core/cloudfront_age.py]: Am I missing something obvious or does this not retry?

File: checks/remotesettings/attachments_availability.py [hunk 2]
```diff
@@ -8,12 +8,13 @@
...
+from poucave.utils import ClientSession
 
 from .utils import KintoClient
 
 
 async def test_url(url):
-    async with aiohttp.ClientSession() as session:
+    async with ClientSession() as session:
         try:
             async with session.head(url) as response:
```
Comment (glasserc) [inline checks/remotesettings/attachments_availability.py]: Does this retry?

File: checks/remotesettings/certificates_expiration.py [hunk 3]
```diff
@@ -29,7 +29,7 @@
 
 
 async def fetch_certificate_expiration(x5u: str) -> datetime:
-    async with aiohttp.ClientSession() as session:
+    async with ClientSession() as session:
         async with session.get(x5u) as response:
             cert_pem = await response.text()
```
Comment (glasserc) [inline checks/remotesettings/certificates_expiration.py]: Could this be `fetch_text`?

**Pull request merged**